icicle-core/src/main/resources/id-generation.lua

```diff
@@ -47,6 +50,7 @@ See the "Scripts as pure functions" section at http://redis.io/commands/eval for
 local time = redis.call('TIME')
 
 return {
+  current_sequence,
```

Comment:
I'm thinking we should rename `current_sequence` and `sequence` to `start_sequence` and `end_sequence`, respectively, and do the `+1` on `start_sequence` in here instead of in the client lib. That way, if somebody wants to write the implementation in another language, they don't have to remember to do the `+1` and the names are a bit more obvious.